frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,76 @@
+import { Flex } from '@mantine/core'
+import { ReactComponent as Logo } from './../img/logo.svg';
+import { ReactComponent as Facebook } from './../img/Facebook.svg';
+import { ReactComponent as Instagram } from './../img/Instagram.svg';
+import { ReactComponent as Youtube } from './../img/Youtube.svg';
+import { ReactComponent as LinkedIn } from './../img/Linkedin.svg';
+
+export const Footer = () => {
+  return (
+    <Flex
+      bg="#9795B5"
```

Comment:
Extract common colors to an Enum like so:

```
export const Color = Object.freeze({
  Primary: "#ffffff",
  Secondary: "#ffffff",
  Accent: "#ffffff",
  Text: "#ffffff",
});
```
This will greatly improve the ease of color modification throughout the app and as an added bonus it will make sure the colors stay consistent no matter where you use them.